src/vs/workbench/contrib/update/browser/releaseNotesEditor.ts

```diff
@@ -288,9 +288,8 @@ export class ReleaseNotesManager {
 					}
 
 					code {
-						font-family: var(--vscode-editor-font-family);
-						font-weight: var(--vscode-editor-font-weight);
-						font-size: var(--vscode-editor-font-size);
+						font-family: Menlo, Monaco, Consolas, "Droid Sans Mono", "Courier New", monospace, "Droid Sans Fallback";
+						font-size: 14px;
```

Comment:
Makes sense for inline code. For codeblocks though, I believe we still should try to use the user's settings. We try to use their theme for codeblocks as well.

Can you confirm this will not effect codeblocks

Reply:
Yes, this will affect code block